src/Components/FormGroup/component.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+import Form from 'react-bootstrap/Form';
+
+type FormProps = {
+    controlId: String,
+    muted?: string,
+    label: String,
+    type: String,
+    placeholder?: string
+};
+
+type FormState = {
+    controlId: String,
+    muted?: string,
+    label: String,
+    type: String,
+    placeholder?: string
+};
+
+class FormGroup extends Component<FormProps, FormState> {
+    state = {
+        controlId: this.props.controlId,
+        muted: this.props.muted,
+        label: this.props.label,
+        type: this.props.type,
+        placeholder: this.props.placeholder
+    };
+
+    render() {
+        return (
+            <Form.Group controlId={this.state.controlId}>
+                <Form.Label>{this.state.label}</Form.Label>
+                <Form.Control
+                    type={this.state.type}
+                    placeholder={this.state.placeholder}
+                />
+
+                <Form.Text className="text-muted">
+                    {this.state.muted !== undefined ? this.state.muted : ''}
+                    {/* why cant we use null? */}
```

Comment:
As I wrote somewhere else, I prefer to use only `undefined` unless 3rd party lib or API returns "null" value. Imo working with `undefined` is easier than with `null`. Do you have any reasons to use `null`?

Reply:
i have usually worked with null but its okay if its a preference issue :D i got no reasons for using it or not using it.

Reply:
Yeah, I believe it comes from the fact that `typeof foo !== 'undefined'` was quite faster than any other kind of `foo !== null` or similar checks in the past. Right now maybe it is not as big issue as it was. But I have a habit now. 

You could find plenty of material on undefined vs null. I believe `undefined` is still more accepted than using `null`, but probably only on preference base nowadays.

Reply:
okay thanks